internal/pkg/aws/elbv2/elbv2.go

```diff
@@ -73,6 +74,27 @@ func (e *ELBV2) ListenerRuleHostHeaders(ruleARN string) ([]string, error) {
 	return hostHeaders, nil
 }
 
+// ListenerRuleIsRedirect returns true if ruleARN has an action to redirect.
+func (e *ELBV2) ListenerRuleIsRedirect(ctx context.Context, ruleARN string) (bool, error) {
```

Comment:
What do you think of?
```suggestion
func (elb *ELBV2) HasRedirectAction(ctx context.Context, ruleARN string) (bool, error) {
```

Alternatively, we can do something like this that's a bit more work but I think a nicer API:
```go
// DescribeRule returns information about a listener rule.
func (elb *ELBV2) DescribeRule(ruleARN string) (Rule, error)

// Rule represents a load balancer listener rule.
type Rule struct { ... }

// HasRedirectAction returns true if the rule has an action that redirects.
func (r Rule) HasRedirectAction() bool
```